src/components/QuranReader/ReadingView/Line.module.scss

```diff
@@ -1,8 +1,16 @@
+@use "src/styles/breakpoints";
+
 .line {
   text-align: center;
 }
 
 .container {
-  display: block;
   direction: rtl;
 }
+
+.inline {
```

Comment:
I don't like the `inline` and `centerText` naming because the behavior is not inline on desktop. 

We can rename it to `mobileInline` and `mobileCenterText` to make the representation more accurate.